Fix Javadoc warnings

### Describe the bug

There are javadoc warnings

### Expected behavior

_No response_

### Minimal reproducible example

_No response_

### Add-on Version

2.6.1

### Vaadin Version

-

### Additional information

```
[WARNING] Javadoc Warnings
[WARNING] J:\fc\addons\ChipFieldAddon\src\main\java\com\flowingcode\vaadin\addons\chipfield\ChipField.java:299: warning: Tag @link: reference not found: #setReadOnly()
[WARNING] /** @deprecated use {@link #setReadOnly()} */

--- a/pom.xml
+++ b/pom.xml
@@ -324,7 +324,7 @@
                         <configuration>
                             <quiet>true</quiet>
                             <doclint>none</doclint>
-                            <failOnWarnings>false</failOnWarnings>
+                            <failOnWarnings>true</failOnWarnings>
                             <links>
                                 <link>https://javadoc.io/doc/com.vaadin/vaadin-platform-javadoc/${vaadin.version}</link>
                             </links>

--- a/src/main/java/com/flowingcode/vaadin/addons/chipfield/ChipField.java
+++ b/src/main/java/com/flowingcode/vaadin/addons/chipfield/ChipField.java
@@ -296,7 +296,7 @@ public class ChipField<T> extends AbstractField<ChipField<T>, List<T>>
     return getElement().getProperty("disabled", false);
   }
 
-  /** @deprecated use {@link #setReadOnly()} */
+  /** @deprecated use {@link #setReadOnly(boolean)} */
   @Deprecated
   public void setReadonly(boolean readonly) {
     setReadOnly(readonly);

